parachain/runtime/snowbridge/src/lib.rs

```diff
@@ -670,6 +670,7 @@ impl_runtime_apis! {
 			add_benchmark!(params, batches, pallet_timestamp, Timestamp);
 			add_benchmark!(params, batches, verifier_lightclient, VerifierLightclient);
 			add_benchmark!(params, batches, assets, Assets);
+			add_benchmark!(params, batches, basic_channel_inbound, BasicInboundChannel);
```

Comment:
Maybe its possible to initialize the locked DOT balance here instead of the chain spec?

```rust
use frame_support::traits::Currency;

Balances::make_free_balance_be(&DotModuleId::get().into_account(), 1000u32.into());

add_benchmark!(params, batches, frame_system, SystemBench::<Runtime>);
add_benchmark!(params, batches, pallet_balances, Balances);	
...
```